scripts/src/java/org/oppia/android/scripts/proto/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+"""
+This library contains all protos used in the scripts module.
+In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
+The proto_library() rule creates a proto file library to be used in multiple languages.
+The java_lite_proto_library() rule takes in a proto_library target and generates java code.
+For more context on adding a new proto library, please refer to model/BUILD.bazel
+"""
+
+load("@rules_java//java:defs.bzl", "java_lite_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+    name = "filename_pattern_validation_structure_proto",
```

Comment:
Prefer naming this & the lite proto library based on the filename, e.g. ``filename_pattern_validation_checks_proto`` (structure is sort of implied with 'proto' in the name).

Reply:
Done.